src/main/java/crazypants/enderio/machine/generator/zombie/TileZombieGenerator.java

```diff
@@ -146,7 +143,7 @@ private boolean generateEnergy() {
 
     //once full, don't start again until we have drained 10 seconds worth of power to prevent
     //flickering on and off constantly when powering a machine that draws less than this produces
-    if(inPause && getEnergyStored() >= (getMaxEnergyStored() - (outputPerTick * 200))) {
+    if (inPause && getEnergyStored() > (getMaxEnergyStored() - (getPowerUsePerTick() * Config.zombieGeneratorResumeTime))) {
```

Comment:
Would it not make sense here to do something like check for less than the '10 seconds anti-chatter' OR 'completely empty'?
`boolean energyLow = (getEnergyStored() > (getMaxEnergyStored() - (getPowerUsePerTick() * Config.zombieGeneratorResumeTime))) || (getEnergyStored() <= 0)
if (inPause && energyLow ) {`

(Note: Possible syntax mistakes, I don't use java on a regular basis).